templates/partials/menu.html

```diff
@@ -16,16 +16,14 @@
                         <a class="p-2 menu-link" aria-current="page" href="index.html">Home</a>
                     </li>
                     <li class="nav-item">
-                        <a class="p-2 menu-link" href="datacatalog.html">Data Catalog</a>
+                        <a class="p-2 menu-link" href="index.html#technology">Technology</a>
                     </li>
-                   <li class="nav-item">
-                        <a class="p-2 menu-link" href="ide.html">IDE</a>
-                    </li>
-                    <li class="nav-item">
-                        <a class="p-2 menu-link" href="remote-processing.html">Remote Processing</a>
-                    </li>
-                    <li class="nav-item">
-                        <a class="p-2 menu-link" href="tutorials.html">Tutorials</a>
+                    <li class="nav-item dropdown">
```

Comment:
This is not what was discussed. We want this link to go to the tutorials page, we do not want a dropdown here.

Reply:
Reverted the changes to the tutorials menu